schemas/src/registry_schemas/schemas/contact_point.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/contactPoint",
+  "required": [
+    "email",
+    "phone"
+  ],
+  "type": "object",
+  "title": "Business Contact Point Schema",
+      "properties": {
+          "email": {
+              "type": "string",
+              "format": "email"
+          },
+          "phone": {
+              "type": "string",
+              "format": "phone"
+          },
+          "extension": {
+              "type": "number"
+          }
+      }
+}
```

Comment:
Nice use of schema.org (JSON LD). Although technically it's "telephone" and not "phone" according to that schema :)